packages/components/src/components/Trigger/Trigger.js

```diff
@@ -37,120 +38,215 @@ const Trigger = ({ intl, eventListenerNamespace, trigger }) => {
+                  if (interceptor.github) {
+                    // GitHub Interceptor
+                    data = interceptor.github;
+                    titlePrefix = intl.formatMessage({
+                      id: 'dashboard.triggerDetails.githubInterceptorName',
+                      defaultMessage: '(GitHub)'
+                    });
+                  } else {
+                    // GitLab Interceptor
+                    data = interceptor.gitlab;
+                    titlePrefix = intl.formatMessage({
+                      id: 'dashboard.triggerDetails.gitlabInterceptorName',
+                      defaultMessage: '(GitLab)'
+                    });
+                  }
+                  const eventTypes = data.eventTypes.join(', ');
+                  title = `${titlePrefix} ${eventTypes}`;
+                  const secretText = intl.formatMessage({
+                    id: 'dashboard.triggerDetails.webhookInterceptorSecret',
+                    defaultMessage: 'Secret:'
+                  });
+                  const secretKeyText = intl.formatMessage({
+                    id: 'dashboard.triggerDetails.webhookInterceptorSecretKey',
+                    defaultMessage: 'Key:'
+                  });
+                  content = (
+                    <>
+                      <p>{secretText}</p>
+                      <div className="interceptor--secret-details">
+                        <p>
+                          {nameText} {data.secretRef.secretName}
+                        </p>
+                        <p>
+                          {secretKeyText} {data.secretRef.secretKey}
+                        </p>
+                        {data.secretRef.namespace && (
+                          <p>
+                            {namespaceText} {data.secretRef.namespace}
+                          </p>
+                        )}
+                      </div>
+                      <p>Event Types: {eventTypes}</p>
+                    </>
+                  );
+                } else if (interceptor.cel) {
+                  // CEL Interceptor
+                  title = `${intl.formatMessage({
+                    id: 'dashboard.triggerDetails.celInterceptorName',
+                    defaultMessage: '(CEL)'
+                  })} ${interceptor.cel.filter}`;
+                  const filter = intl.formatMessage({
+                    id: 'dashboard.triggerDetails.celInterceptorFilter',
+                    defaultMessage: 'Filter: '
+                  });
+                  content = (
+                    <>
+                      <p>{filter}</p>
+                      <code className="bx--snippet--multi interceptor--cel-filter">
+                        {interceptor.cel.filter}
+                      </code>
+                    </>
+                  );
+                } else {
+                  return null;
+                }
+                return (
+                  <AccordionItem key={title} title={`${index + 1}. ${title}`}>
```

Comment:
we can't concatenate strings like this as the order may be different in some languages (e.g. RTL languages such as Arabic / Hebrew)

instead we should use a placeholder in the string for the value we're going to substitute

e.g.

```js
intl.formatMessage(
  {
    id: 'dashboard.triggerDetails.celInterceptorTitle',
    defaultMessage: '(CEL) {interceptorName}'
  },
  {
    interceptorName
  }
)
```